backend/src/schema/repository.ts

```diff
@@ -63,6 +65,18 @@ const resolvers = {
       const branchName = await getCurrentBranchName(repoLocation)
       return { branchName: branchName }
     },
+    getRepoBranches: async (
+      _root: unknown,
+      args: { url: string },
+      _context: unknown
+    ): Promise<string[]> => {
+      const repoUrl = new URL(args.url)
+      const repositoryName = repoUrl.pathname
+      const repoLocation = `./repositories/${repositoryName}`
+      const branches = await getBranches(simpleGit(repoLocation))
+
+      return branches
```

Comment:
Would refactor either this or `getCurrentBranchName` so that simplegit is imported in only one place. Not a blocker for this PR.